addons/fleet/models/fleet_vehicle.py

```diff
@@ -93,13 +94,24 @@ def _get_default_state(self):
     plan_to_change_bike = fields.Boolean(related='driver_id.plan_to_change_bike', store=True, readonly=False)
     vehicle_type = fields.Selection(related='model_id.vehicle_type')
     frame_type = fields.Selection([('diamant', 'Diamant'), ('trapez', 'Trapez'), ('wave', 'Wave')], help="Frame type of the bike")
-    electric_assistance = fields.Boolean()
+    electric_assistance = fields.Boolean(compute='_compute_model_fields', store=True, readonly=False)
     frame_size = fields.Float()
 
-    @api.depends('vehicle_type')
-    def _compute_doors(self):
-        for record in self:
-            record.doors = 5 if record.model_id.vehicle_type == 'car' else 0
+    @api.depends('model_id')
```

Comment:
Shouldn't we use instead:
```suggestion
    @api.depends(lambda self: ['model_id.%s' % (key) for key in MODEL_FIELDS_TO_VEHICLE.keys()])
```
?

Reply:
I believe it's better with just model_id, imagine you change the number of doors on `model_id` you would not want that change to be applied to existing vehicles, model_id acts more like a preset in this case

Reply:
Right ! :)